packages/router/useRouteParams/index.ts

```diff
@@ -64,7 +64,9 @@ export function useRouteParams<
         trigger()
 
         nextTick(() => {
-          router[toValue(mode)]({ ...route, params: { ...route.params, ...Object.fromEntries(_params.entries()) } })
+          const { name: pathName, path, ...otherRouteParams } = route
+          const routeParams = { name: pathName, ...otherRouteParams }
+          router[toValue(mode)]({ ...routeParams, params: { ...route.params, ...Object.fromEntries(_params.entries()) } } as RouteLocationRaw)
```

Comment:
Hi @Tzyito , We don't need the other props from the route. What do you think about using just what we need?

```ts
        nextTick(() => {
          const { params, query, hash } = route;

          router[toValue(mode)]({
            params: {
              ...params,
              ...Object.fromEntries(_params.entries()),
            },
            query,
            hash,
          });
        });
```

Reply:
@anteriovieira oh,sorry.i think you are right

Reply:
no problem, I'll merge as soon as you update